plugins/scaffolder-backend/src/scaffolder/stages/templater/helpers.test.ts

```diff
@@ -126,6 +126,25 @@ describe('helpers', () => {
       );
     });
 
+    it('should pass through the user and group id from the host machine', async () => {
+      await runDockerContainer({
+        imageName,
+        args,
+        templateDir,
+        resultDir,
+        dockerClient: mockDocker,
+      });
+
+      expect(mockDocker.run).toHaveBeenCalledWith(
+        imageName,
+        args,
+        expect.any(Stream),
+        expect.objectContaining({
+          User: `${process.getuid()}:${process.getgid()}`,
```

Comment:
Is it not the other way around? (And happens to work because they often are the same)

Just checking

Reply:
https://docs.docker.com/engine/reference/commandline/exec

Name, shorthand | Default | Description
-- | -- | --
--user , -u |   | Username or UID (format: <name\|uid>[:<group\|gid>])

glad I got it right! 🙌